### ISSUE TITLE
[Issue]: Some m4a files with chapters do not show cover art

### ISSUE THREAD

**Issue by tscheney** [jellyfin/jellyfin#9131]
### Please describe your bug

Some m4a files do not shown cover art when the m4a file also includes chapters.  So far I haven't been able to figure out why some with chapters do show, but in every case removing the chapters causes the cover to show. I've attached a couple of files that I generated. They are exactly the same except one has chapters in it and the other I cleared the chapters by running a ffmpeg -acodec copy -vcodec copy -map_chapters -1 on it. Jellyfin will not show the cover art for the one with the chapters (cover not shown.m4a), but it does for the one without (cover shown.m4a).
[cover shown.zip](https://github.com/jellyfin/jellyfin/files/10463197/cover.shown.zip)

[cover not shown.zip](https://github.com/jellyfin/jellyfin/files/10463191/cover.not.shown.zip)

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
Please check if the bug still exists in 10.8

**Comment by tscheney** [comment]
Hi, thanks for the reply. I updated to 10.8.8, and it appears the issue is solved. However, I tried updating via Watchtower because I have Jellyfin running in a Docker container, but for some reason it wiped out everything. Everything appears to be working now, so I'm guessing that upgrading fixed it, but unfortunately I can't say for sure if it was that or just re-creating my entire database that did it.
